### PR TITLE
Addition of CIS Azure benchmark type

### PR DESCRIPTION
**Pull request by jeniawhite**
### Summary of your changes
Added CIS Azure benchmark type with config in order to support the new benchmark

### Related Issues
- Fixes: #1290 

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

**Review by orestisfl — COMMENTED** [review]
Can you add a test case in `config_test.go` to confirm the expected behavior?

**Review by oren-zohar — APPROVED** [review]

**Review by uri-weisman — APPROVED** [review]
cloudbeat-azure.yml LGTM

File: deploy/azure/cloudbeat-azure.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+cloudbeat:
+  config:
+    v1:
+      benchmark: cis_azure
+  type: cloudbeat/cis_azure
+  # Defines how often an event is sent to the output
+  period: 30s
```
Comment (gurevichdmitry) [inline deploy/azure/cloudbeat-azure.yml:7]: Will we use Azure credentials, and should we configure them?

Comment (orestisfl) [reply deploy/azure/cloudbeat-azure.yml:7]: Probably, but let's track this in https://github.com/elastic/cloudbeat/pull/1308

Comment (gurevichdmitry) [reply deploy/azure/cloudbeat-azure.yml:7]: thanks, cloudbeat-azure.yml LGTM

**Review by gurevichdmitry — APPROVED** [review]
LGTM

**Pull request merged**